application/pom.xml

```diff
@@ -286,7 +286,6 @@
         <dependency>
             <groupId>com.jayway.jsonpath</groupId>
             <artifactId>json-path</artifactId>
-            <scope>test</scope>
```

Comment:
Hi @YuriyLytvynchuk, if I'm not mistaken, we don't need to remove the test scope in application module.